rfcs/0001-messages.md

```diff
@@ -0,0 +1,146 @@
+# FROST messages
+
+Proposes a message layout to exchange information between participants of a FROST setup.
+
+## Motivation
+
+Currently FROST library is complete for 2 round signatures with a dealer/aggregator setup.
```

Comment:
Are these messages intended to be generic, or specific to a particular signature scheme?